stdlib/public/runtime/Errors.cpp

```diff
@@ -231,6 +235,9 @@ reportNow(uint32_t flags, const char *message)
 #ifdef __APPLE__
   asl_log(nullptr, nullptr, ASL_LEVEL_ERR, "%s", message);
 #endif
+#ifdef __ANDROID__
+  __android_log_print(ANDROID_LOG_ERROR, "Swift", "%s", message);
```

Comment:
I wondered about ERROR v. FATAL. I’ve updated it and changed to a #elif. For the “tag” it is displayed as prefix Swift/F before the message. Ideally It would be the process/image name but I couldn’t find a portable way to determine it in the NDK. I’ve changed it to “SwiftRuntime”.